### PR TITLE
Edge Bridge Tutorial Update - final app, doc and asset cleanup

### PR DESCRIPTION
**Pull request by timkimadobe**
## Description

Tutorial document related changes:
- Transfer of standalone data mapper tutorial information into Edge Bridge tutorial and subsequent removal
- Cleanup of tutorial document and organization of assets by tutorial section
- Addition of assurance validation screenshot

Tutorial App related changes:
- Creating final version of tutorial app
- Removing test files from start and final app
- Cleanup of code block comments

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by cacheung — APPROVED** [review]

**Comment by emdobrin** [comment]
same question here, can we remove map-track-data-using-data-prep.md since the content is already covered in the tutorial?
additionally can you include a quick table of content readme at top level?

File: Documentation/tutorials/EdgeBridgeTutorialAppFinal/app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    // Add dependencies on AEP Mobile extensions
+    implementation 'com.adobe.marketing.mobile:assurance:1.0.4'
+    implementation 'com.adobe.marketing.mobile:core:1.11.4'
+    implementation 'com.adobe.marketing.mobile:lifecycle:1.1.1'
+    implementation 'com.adobe.marketing.mobile:edge:1.3.2'
+    implementation 'com.adobe.marketing.mobile:edgeconsent:1.0.1'
+    implementation 'com.adobe.marketing.mobile:edgeidentity:1.1.0'
+
+    //* Edge Bridge Tutorial - code section (1/2)
+    implementation fileTree(dir: 'libs', include: ['edgebridge-release-1.0.0-beta.aar'])
```
Comment (emdobrin) [inline Documentation/tutorials/EdgeBridgeTutorialAppFinal/app/build.gradle:53]: keep a note that we need to update this after the staging build is created

**Review by emdobrin — COMMENTED** [review]
Can you please include the same feedback from the ios tutorial here?

File: Documentation/tutorials/edge-bridge-tutorial.md [hunk 2]
```diff
@@ -190,6 +191,9 @@ With the block uncommented, you should see a blue ribbon appear at the top of th
 
 Gradle will use the configuration settings we just activated to install all of the Edge Bridge extensions we want for our tutorial app, and allow us to use their features within the app's code.
 
+> **Note**  
+> After this step, there are no Adobe Experience Cloud Solution extensions in the app. At this point, the import and registration for the AEPIdentity extension can also be removed since AEPEdgeIdentity takes care of the identity functionality for the Edge extensions.
+> If your application still uses Adobe Experience Cloud Solution extensions, such as Adobe Target, Adobe Campaign, etc. (find the full list [here](https://aep-sdks.gitbook.io/docs/)) you should ignore the steps below for removing AEPIdentity and continue to use the extension.
```
Comment (emdobrin) [inline Documentation/tutorials/edge-bridge-tutorial.md:196]: Some renaming for Android:
AEPIdentity -> Identity
AEPEdgeIdentity -> Identity for Edge Network

Comment (emdobrin) [reply Documentation/tutorials/edge-bridge-tutorial.md]: nit: one more at line 196

**Review by emdobrin — APPROVED** [review]

**Pull request merged**